app/grants/migrations/0036_merge_20191219_0819.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 2.2.4 on 2019-12-19 08:19
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('grants', '0035_grant_category'),
+        ('grants', '0035_auto_20191214_0736'),
+    ]
+
+    operations = [
+    ]
```

Comment:
could we remove the all the migrations and recreate it with just one migration file ?

Reply:
Sure, that makes sense